src/makeCalculator.js

```diff
@@ -5,6 +5,49 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add: function (num) {
+      this.result += num;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - When defining methods within an object, you can use the shorthand method syntax. Replace the `function` keyword with the shorthand syntax.